src/commands/windows/lastshutdown.rs

```diff
@@ -0,0 +1,82 @@
+use std::collections::HashMap;
+
+use clap::Command as ClapCommand;
+use windows::{core::*, Win32::System::Variant::VARIANT};
+
+use bitconv::{endian::Little, to_uint64};
+use chrono::DateTime;
+
+use crate::{
+    commands::base::registry::CommandRegistration,
+    commands::base::{
+        Command, CommandDTO, CommandData,
+        CommandResult::{self, Simple},
+    },
+    runtime::Runtime,
+    utils::registry::{get_binary_value, RegistryHive},
+};
+
+pub struct LastShutdownCommand {
+    data: CommandData,
+}
+
+inventory::submit! {
+    CommandRegistration {
+        name: "lastshutdown",
+        factory: || Box::new(LastShutdownCommand::default()),
+        clap_command: || ClapCommand
+            ::new("lastshutdown")
+            .version("1.0")
+            .about("A command that check the last shutdown of the local computer")
+    }
+}
+
+impl Command for LastShutdownCommand {
+    fn execute(&self, runtime: &Runtime, _: &[String]) -> Result<CommandResult> {
+
+        // Get the shutdown bytes form the registry. These bytes represent the Windows FileTime.
+        let shutdown_bytes = get_binary_value(
+            RegistryHive::LocalMachine,
+            "SYSTEM\\ControlSet001\\Control\\Windows",
+            "ShutdownTime"
+        )?;
+
+        // Difference between Windows FileTime Epoch (January 1, 1601 UTC) and Unix Time Epoch (January 1, 1970 UTC)
+        const EPOCH_DIFFERENCE: u64 = 116444736000000000;
```

Comment:
Why is Microsoft so Microsoft? Who came up with this?? 💀 

Learned something new tho ❤️